js/ui/html_editor/ui.html_editor.js

```diff
@@ -283,13 +283,16 @@ const HtmlEditor = Editor.inherit({
     },
 
     _applyTranscludedContent: function() {
-        const markup = this._deltaConverter.toHtml();
-        const newDelta = this._quillInstance.clipboard.convert(markup);
-
-        if(newDelta.ops.length) {
-            this._quillInstance.setContents(newDelta);
-        } else {
-            this._finalizeContentRendering();
+        const valueOption = this.option('value');
+        if(valueOption === undefined || valueOption === null) {
+            const markup = this._deltaConverter.toHtml();
+            const newDelta = this._quillInstance.clipboard.convert(markup);
+
+            if(newDelta.ops.length) {
+                this._quillInstance.setContents(newDelta);
+            } else {
+                this._finalizeContentRendering();
+            }
         }
```

Comment:
```suggestion
        } else {
            this._finalizeContentRendering();
        }
```

Check that `contentReady` event works fine with this case